cypress/src/salesforce/contact/new-contact.js

```diff
@@ -0,0 +1,117 @@
+const contact = require("../../../fixtures/locator/contact/new-contact.json");
+
+    export function addContactSalutation(salutation) {
```

Comment:
I think this tab should not be there, because this function is not the child of any other function